test/unit/object/test_folder.py

```diff
@@ -167,3 +167,19 @@ def test_update_sync_state(test_folder, mock_folder_response, mock_box_session,
     mock_box_session.put.assert_called_once_with(expected_url, data=json.dumps(data), params=None, headers=None)
     assert isinstance(update_response, Folder)
     assert update_response.object_id == test_folder.object_id
+
+
+def test_preflight(test_folder, mock_object_id, mock_box_session):
```

Comment:
Any suggestions about exactly how to parametrize size/name? I did it for preflight in File because 'name' can be omitted.